backend/functions/routes/coords.js

```diff
@@ -0,0 +1,76 @@
+// Set up Firebase functions
+const functions = require('firebase-functions');
+
+// Set up Firebase Admin for Firestore
+const admin = require('firebase-admin');
+admin.initializeApp(functions.config().firebase);
+let db = admin.firestore();
+
+// Set up Express
+const express = require('express');
+const app = express();
+const cors = require('cors');
+app.use(cors({origin: true})); // Automatically allow cross-origin requests
+
+// Coordinates paths
+const COLLECTION = 'coordinates';
+const CREATE_COORD = '/create';
+const READ_COORD = '/read';
```

Comment:
No need to specify CRUD actions into endpoint. We should use HTTP method verbs to express what that endpoind does with our request.

Reply:
This was the only clear action words I could come up with. Didn't want any ambiguous words.